pages/clubs.jsx

```diff
@@ -409,6 +409,7 @@ var ModalAddOrChangeYourClub = React.createClass({
              </div>
            : null}
           {this.renderValidationErrors()}
+          <p>We're full! Let us know if you'd like to be considered for the next round.</p>
```

Comment:
Oh good question. I think that typographically, using `&lsquo;` is nice because it renders as a "curly" quote rather than a straight one. It does make the source code harder to read, though, which is annoying... An alternative would be to use the actual unicode character, since I believe all our JS and JSX files are interpreted as UTF-8. Then it's just a matter of figuring out how the heck to type such a character on our keyboards, lol.

I guess I'd actually defer to @sabrinang on this one... We could either be consistent about it and actually use smart quotes and other nice typographic conventions wherever possible, or we could just be opportunistic about it and use it whenever it's easy and we happen to remember to do it (which is sort of what we're doing now), or we could opt for consistency in the opposite direction and *never* use smart quotes and other niceties.